packages/backend/src/database/migrations/20230628153935_allow-email-domains-projects-table.ts

```diff
@@ -0,0 +1,112 @@
+import { Knex } from 'knex';
+
+const OrganizationAllowedEmailDomainProjectsTableName =
+    'organization_allowed_email_domain_projects';
+const OrganizationAllowedEmailDomainsTableName =
+    'organization_allowed_email_domains';
```

Comment:
can't we add a new column (project_uuid) on the existing table ?

Reply:
there are multiple projects per allowed email domains, that's why we needed the separate table.